TuberculosisApp/app/src/main/java/com/rugged/tuberculosisapp/network/RetrofitClientInstance.java

```diff
@@ -5,7 +5,7 @@
 
 public class RetrofitClientInstance {
     private static Retrofit retrofit;
-    private static final String BASE_URL = "http://192.168.50.4:2002/api/"; // TODO change when database is stored on server
+    private static final String BASE_URL = "http://192.168.178.47:2002/api/"; // TODO change when database is stored on server
```

Comment:
Did the url change for the database?

Reply:
I used this ip to test the API functionality on my phone while hosting the server on my pc, the other syntax didnt fit my router's default gateway pattern (192.168.178.x)